shared/templates/firefox_policy-setting/template.py

```diff
@@ -0,0 +1,131 @@
+from __future__ import print_function
+import re
+import json
+
+
+def normalize_policy_path(path_string):
+    """Generate a normalized path array from yaml.
+       Paths are space-separated key names.
+       All firefox policies are under the policies key.
+    """
+    path = []
+    if len(path_string.strip()) > 0:
+        path = re.split(" ", path_string)
+    if 'policies' not in path:
+        path = ['policies'] + path
+    return path
+
+
+def make_json_boolean(instr, oval_format=False):
```

Comment:
Those are basically two different functions, one when `oval_format` is true, and the other one when it is false. So please make it into two functions.

Next, argument `instr` is not descriptive, and it is also not clear what should be the output - one code path returns actual boolean, one returns a regular expression, and other return a string. So the function name is also questionable. Please try to make also the function name descriptive.

Reply:
I must have misunderstood [you before](https://github.com/ComplianceAsCode/content/pull/8271#discussion_r872481374) then because I *did* have those two functions for rendering JSON booleans separate.

Reply:
Probably, "split" means that one should end up with more functions. Generally, this is a good direction.

Reply:
https://github.com/ComplianceAsCode/content/pull/8271/commits/cb89dc72a97d03ab77febe9e2552d8fee049bf7f